src/translations/fi.json

```diff
@@ -57,9 +57,12 @@
   "page.berths.list.berth_count": "{count} hakuehdot täyttävää satamaa",
   "page.berths.list.header.hits": "Hakukriteerit täyttävät satamat:",
   "page.berths.list.header.others": "Muut satamat:",
-  "page.berths.status.green": "Vähän jonoa",
-  "page.berths.status.yellow": "Jonkin verran jonoa",
-  "page.berths.status.red": "Paljon jonoa",
+  "page.berths.status.title.green": "Vähän jonoa",
+  "page.berths.status.description.green": "Vähän jonoa",
```

Comment:
seems interesting... same translation 🤔

Reply:
Because it should be the same, only red status has different tooltip value that what is shown on page.